stdlib/sys/process.gr

```diff
@@ -94,6 +94,14 @@ provide let argv = () => {
   }
 
   let argc = WasmI32.load(argcPtr, 0n)
+
+  let argsLength = argc * 4n
+  let arr = allocateArray(argc)
+
+  if (WasmI32.eqz(argsLength)) {
+    return Ok(WasmI32.toGrain(arr): Array<String>)
```

Comment:
```suggestion
    Memory.free(argcPtr)
    return Ok(WasmI32.toGrain(arr): Array<String>)
```
could also refactor a bit to do the free after we read `argc` so we don't need to do it in multiple places.

Reply:
I tried to do the refactor. Let me know if it is correct!